mail/src/main/scala/spinoco/protocol/mail/imap/codec/IMAPBodyPartCodec.scala

```diff
@@ -95,8 +95,9 @@ object IMAPBodyPartCodec {
       }
 
       choice(
-        DQUOTE ~> date <~ DQUOTE
-        , date
+        NIL.xmap[None.type](_ => None, _ => ()).upcast[Option[LocalDate]]
```

Comment:
This will always write a the date as Nil, that does not seem correct, this should be last line.